src/renderer/app.js

```diff
@@ -0,0 +1,49 @@
+import {ipcRenderer, shell} from 'electron';
```

Comment:
Note that the contents of this file, `app.js`, are identical to the old `index.js`. Only the name has changed to ~protect~ help with loading the file asynchronously.